scripts/check-alembic-revision.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import os
+import ast
+
+
+def check_alembic_file_for_pass(file_path):
+    if not os.path.isfile(file_path):
+        print(f"File not found: {file_path}")
+        return
+
+    with open(file_path, "r") as file:
+        tree = ast.parse(file.read(), filename=file_path)
+
+    upgrade_has_pass = False
+    downgrade_has_pass = False
+
+    for node in ast.walk(tree):
+        if isinstance(node, ast.FunctionDef):
+            if node.name == "upgrade":
+                for stmt in node.body:
+                    if isinstance(stmt, ast.Pass):
+                        upgrade_has_pass = True
+            elif node.name == "downgrade":
+                for stmt in node.body:
+                    if isinstance(stmt, ast.Pass):
+                        downgrade_has_pass = True
```

Comment:
We should check if the only valid statement is `pass`, not just inclusion.

For instance,
```python
def upgrade():
    if something:
        pass
    else:
        some_operation()
```
could also satisfy this logic.